testsuite/manualmode/src/test/java/org/wildfly/core/test/standalone/mgmt/PreparedResponseTestCase.java

```diff
@@ -0,0 +1,152 @@
+import org.jboss.logging.Logger;
+import org.jboss.msc.service.ServiceActivator;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.exporter.ZipExporter;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.core.testrunner.ManagementClient;
+import org.wildfly.core.testrunner.ServerControl;
+import org.wildfly.core.testrunner.ServerController;
+import org.wildfly.core.testrunner.WildflyTestRunner;
+import org.wildfly.test.shutdown.SlowStopService;
+import org.wildfly.test.shutdown.SlowStopServiceActivator;
+
+/**
+ *
+ * @author <a href="mailto:ehugonne@redhat.com">Emmanuel Hugonnet</a> (c) 2015 Red Hat, inc.
+ */
+@RunWith(WildflyTestRunner.class)
+@ServerControl(manual = true)
+public class PreparedResponseTestCase {
+
+    public static Logger LOGGER = Logger.getLogger(PreparedResponseTestCase.class);
+
+    public static final String SLOW_STOP_JAR = "slow-stop.jar";
+    private static final long FREQUENCY = 500L;
+
+    @Inject
+    protected static ServerController controller;
+
+    @BeforeClass
+    public static void startAndSetupContainer() throws Exception {
+        controller.start();
+        ManagementClient managementClient = controller.getClient();
+        deploy(managementClient);
+    }
+
+    @AfterClass
+    public static void stopContainer() throws Exception {
+        undeploy(controller.getClient());
+        controller.stop();
+    }
+
+    public static void deploy(ManagementClient managementClient) throws Exception {
+        ServerDeploymentHelper helper = new ServerDeploymentHelper(managementClient.getControllerClient());
+        JavaArchive war = ShrinkWrap.create(JavaArchive.class, SLOW_STOP_JAR);
+        war.addPackage(SlowStopService.class.getPackage());
+        war.addAsServiceProvider(ServiceActivator.class, SlowStopServiceActivator.class);
+        war.addAsResource(new StringAsset("Dependencies: org.jboss.dmr, org.jboss.logging, org.jboss.as.controller, org.jboss.as.server,org.wildfly.extension.request-controller, org.jboss.as.network\n"), "META-INF/MANIFEST.MF");
+        helper.deploy(SLOW_STOP_JAR, war.as(ZipExporter.class).exportAsInputStream());
+    }
+
+    public static void undeploy(ManagementClient managementClient) throws ServerDeploymentHelper.ServerDeploymentException {
+        ServerDeploymentHelper helper = new ServerDeploymentHelper(managementClient.getControllerClient());
+        helper.undeploy(SLOW_STOP_JAR);
+    }
+
+    @Test
+    public void reloadServer() throws Exception {
+        ManagementClient mgmtClient = getManagementClient();
+        long timeout = SHUTDOWN_WAITING_TIME + System.currentTimeMillis();
+        mgmtClient.executeForResult(Operations.createOperation(RELOAD, new ModelNode().setEmptyList()));
```

Comment:
In this case I don't think it matters, but I think in general as we write new tests with operations we shouldn't use the constants. It's mainly a regression check to ensure the operation names don't change in the constant value.... ...though I should probably bring this up on wildfly-dev and not this PR :)